fn main() -> compile_fail_utils::ui_test::Result<()> {
    compile_fail_utils::test("reflect_into_closure", "tests/into_closure")
}
